blivet/threads.py

```diff
@@ -33,7 +34,7 @@
 
 def exclusive(m):
     """ Run a callable while holding the global lock. """
-    @wraps(m)
+    @wraps(m, set(functools.WRAPPER_ASSIGNMENTS) & set(dir(m)))
```

Comment:
So ``six.wraps()`` does something else than Python 3's ``functools.wraps()``? What's the point of ``six.wraps()`` then?